cli/dcoscli/main.py

```diff
@@ -89,43 +91,57 @@ def _main():
     subproc = Popen([executable,  command] + args['<args>'],
                     stderr=PIPE)
 
+    if log_level:
```

Comment:
yes, but if they set it in that process shouldn't we keep logging on for that process?